email_composer/src/containers/Composer.js

```diff
@@ -202,6 +204,15 @@ class ComposerWrapper extends Component {
       })
       .filter(alias => alias.isAliasActive);
     state = { ...state, accounts: [...accounts, ...aliasAccounts] };
+    if (fromAddress) {
```

Comment:
You can just get fromAddress from `composerData`. You never change the value of `fromAddress`

Reply:
`composerData` is not in the scope

Reply:
you are right